libs/dh/market-participant/grid-areas/shell/src/lib/dh-grid-areas-shell.component.ts

```diff
@@ -14,50 +14,52 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { NgIf } from '@angular/common';
-import { Component } from '@angular/core';
-import { RxPush } from '@rx-angular/template/push';
-import { RxLet } from '@rx-angular/template/let';
-import { TranslocoDirective } from '@ngneat/transloco';
-
-import { DhMarketParticipantGridAreaOverviewDataAccessApiStore } from '@energinet-datahub/dh/market-participant/data-access-api';
-import { WattValidationMessageComponent } from '@energinet-datahub/watt/validation-message';
-import { WattSpinnerComponent } from '@energinet-datahub/watt/spinner';
-import { WattEmptyStateComponent } from '@energinet-datahub/watt/empty-state';
-
-import { DhMarketParticipantGridAreaOverviewComponent } from './overview/dh-market-participant-gridarea-overview.component';
+import { Component, OnDestroy, OnInit, inject } from '@angular/core';
+import {
+  DhMarketParticipantGridAreaOverviewComponent,
+  GridAreaOverviewRow,
+} from '@energinet-datahub/dh/market-participant/grid-areas/overview';
+import { Apollo } from 'apollo-angular';
+import { GetGridAreaOverviewDocument } from '@energinet-datahub/dh/shared/domain/graphql';
+import { Subject, takeUntil } from 'rxjs';
 
 @Component({
   selector: 'dh-grid-areas-shell',
-  styles: [
-    `
-      .spinner-container {
-        display: flex;
-        justify-content: center;
-      }
-    `,
-  ],
   templateUrl: './dh-grid-areas-shell.component.html',
-  providers: [DhMarketParticipantGridAreaOverviewDataAccessApiStore],
   standalone: true,
-  imports: [
-    NgIf,
-    RxLet,
-    RxPush,
-    TranslocoDirective,
-
-    WattEmptyStateComponent,
-    WattSpinnerComponent,
-    WattValidationMessageComponent,
-    DhMarketParticipantGridAreaOverviewComponent,
-  ],
+  imports: [DhMarketParticipantGridAreaOverviewComponent],
 })
-export class DhGridAreasShellComponent {
-  constructor(private store: DhMarketParticipantGridAreaOverviewDataAccessApiStore) {
-    this.store.init();
+export class DhGridAreasShellComponent implements OnInit, OnDestroy {
+  private readonly gln = new RegExp('^[0-9]+$');
+  private apollo = inject(Apollo);
+  private destroy = new Subject<void>();
```

Comment:
Should probably use "private destroyRef = inject(DestroyRef);"

Reply:
fixed